src/stories/Spacing.stories.tsx

```diff
@@ -8,31 +10,31 @@ type SizeProps = {
 
 export const SpacingBlocks = () => {
   const sizes: SizeProps = {
-    xl: {
+    SpaceXl: {
       color: '--color-green-50',
       mqBase: '32',
       mqSmall: '48',
       mqMedium: '64'
     },
-    lg: {
+    SpaceLg: {
       color: '--color-blue-50',
       mqBase: '16',
       mqSmall: '24',
       mqMedium: '32'
     },
-    md: {
+    SpaceMd: {
       color: '--color-orange-50',
       mqBase: '8',
       mqSmall: '12',
       mqMedium: '16'
     },
-    sm: {
+    SpaceSm: {
       color: '--color-red-50',
       mqBase: '6',
       mqSmall: '8',
       mqMedium: '8'
     },
-    xs: {
+    SpaceXs: {
       color: '--color-amber-40',
       mqBase: '4',
       mqSmall: '4',
```

Comment:
Now I'm looking at this with the context of my comment at https://github.com/wellcometrust/design-system/pull/8#discussion_r684179857... would it not be more appropriate to add a JSON file as a single source of truth for these values, which could serve as a temporary design token file?

That way it would be easy to use those values wherever else in this repo, and would be easier to port to (e.g.) style dictionary later? As far as I understand it would necessitate less code changes, too, should any of the values change (e.g. spaceXs gets 1px larger, a new variable 'spaceXxs' is added).

Reply:
Ultimately there will be a single source of truth but that won't be this object (which is used solely for this documentation) but a token JSON file - [see example here](https://github.com/wellcometrust/design-system/pull/2/files#diff-703b1c57d8c36e3261c7a5a414e550f17ae43e611f6dfe00d7d1a01594f85ac2). The token file is imported into Style Dictionary and at least two files will be output; a CSS file and a JS file containing an object literal (to replace this object).

It's as easy to leave this in place (for now at least) to keep context.

Reply:
Cool, all sounds good to me, just wanted to make sure I was on the same page